libs/test-utils/src/auth.ts

```diff
@@ -5,14 +5,15 @@ import {
   ElectionManagerUser,
   PollWorkerUser,
   SystemAdministratorUser,
+  TEST_JURISDICTION,
 } from '@votingworks/types';
 
 export function fakeSystemAdministratorUser(
   props: Partial<SystemAdministratorUser> = {}
 ): SystemAdministratorUser {
   return {
     role: 'system_administrator',
-    jurisdiction: 'jurisdiction',
+    jurisdiction: TEST_JURISDICTION,
```

Comment:
I realized that the jurisdiction on fake user objects created by `@votingworks/test-utils` wasn't matching the jurisdiction we use in app tests (`DEV_JURISDICTION`). Not a big deal since we're mocking auth in app tests and the jurisdiction mismatch won't actually be detected but did feel a bit misleading. Also made it hard to know which jurisdiction to expect on user objects when writing test expectations.

Through this commit and the next one, I introduce a `TEST_JURISDICTION`, separate from the `DEV_JURISDICTION`, and consistently use it across app tests.